.kontinuous/env/preprod/values.yaml

```
@@ -15,5 +24,14 @@ app:
     MAILER_SMTP_LOGIN: ""
     MAILER_SMTP_PASSWORD: ""
     MAILER_SMTP_SSL: "False"
+  autoscale:
+    enabled: true
+  resources:
+    requests:
+      cpu: 20m
+      memory: 640M
```

```suggestion
      memory: 196M
```

il me semble que c'est démesuré pour le app qui consomme pas plus de 180Mo